packages/grafana-data/src/utils/featureToggle.ts

```diff
@@ -0,0 +1,16 @@
+import { BootData } from '../types';
+
+declare global {
+  interface Window {
+    grafanaBootData?: BootData;
```

Comment:
is this ever not defined? making it required would remove a lot of optional chaining elsewhere 🤔

Reply:
My main concern was that it could be undefined in tests. But maybe I'm being overly cautious.

Reply:
@oscarkilhed ah that's a fair concern 👍 

we do already mock out the bootdata in unit tests here: https://github.com/grafana/grafana/blob/oscark/fix-window-as-any-when-accessing-feature-toggles/public/test/jest-setup.ts#L24, so might just be a case of adding a `featureToggles` object here as well